LayoutTests/fast/forms/switch/no-pixels-outside-the-box-expected.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
+<style>
+body { zoom: 5; display: grid }
+span { display: inline-block; background: blue }
+input, span { grid-area: 1/1 }
+</style>
+
+<input type=checkbox switch>
```

Comment:
I would remove this from the ref?

Reply:
It's important to determine the size.

Reply:
I wonder if we could just use a white overlay and have the ref be a blank file?

Reply:
Nice, will do that.